src/utils/topic.py

```
@@ -0,0 +1,199 @@
+    num_vectors = vdb.index.ntotal
+    qsize = len(ids)
+
+    embs = np.empty((qsize, d), dtype=np.float32)
+    vdb.index.reconstruct_n(num_vectors - qsize, qsize, embs)
+
+    # Get topic connections
+    min_cosine = 0.3
+
+    D, I = vdb.index.search(embs, k=200)
+
+    for row_idx, (distances, neighbors) in enumerate(zip(D, I)):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)